.github/workflows/auto-review-and-pr-summary.yml

```
@@ -84,8 +81,9 @@ jobs:
 
       - name: Update PR body
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_BODY: ${{ steps.generate_pr_body.outputs.pr_body }}
         run: |
           echo "Updating PR body..."
-          gh pr edit ${{ github.event.pull_request.number }} --body "${{ steps.generate_pr_body.outputs.pr_body }}"
-        env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          gh pr edit ${{ github.event.pull_request.number }} --body "$PR_BODY"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)